build.gradle

```diff
@@ -7,7 +7,7 @@ import org.opensearch.gradle.test.RestIntegTestTask
 
 buildscript {
     ext {
-        opensearch_version = System.getProperty("opensearch.version", "2.1.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "2.2.0-SNAPSHOT")
```

Comment:
Main should be on 3.0, no? Either way I would make this change separately.

Reply:
https://github.com/opensearch-project/job-scheduler/pull/206/files

Reply:
Thanks @dblock , I think for most plugin `main` is kept in sync with next OpenSearch release (it seems like `job-scheduler` has not released `2.1.0.0`)

Reply:
See https://github.com/opensearch-project/opensearch-plugins/issues/142, I think it's happening, cc: @prudhvigodithi